### PR TITLE
update conda-forge install procedure

### PR DESCRIPTION
**Pull request by UnixJunkie**
#### What does this implement/fix? Explain your changes.

people were complaining on a mailing list about the current way to install rdkit

### CODE REVIEW

File: Docs/Book/Install.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ C:\> activate my-rdkit-env
 A [conda-forge](https://conda-forge.org/#about) RDKit package is also available, which may offer an easier installation for users already using other conda-forge packages. This package can be installed with:
 
 ```shellsession
-$ conda install -c conda-forge rdkit
+$ conda create -c conda-forge -n rdkit rdkit python=3.8 cairo
+$ conda activate rdkit
```
Comment (greglandrum) [inline Docs/Book/Install.md:48]: This is actually doing something different.
The original command adds rdkit to an existing environment, while the new one creates a new environment.
I'm happy to add the new command in addition to what's already there, but please add a bit more text documenting that this creates a new environment.

I would also suggest modifying the command as follows:
```suggestion
$ conda create -c conda-forge -n py38_rdkit rdkit python=3.8
$ conda activate py38_rdkit
```

**Review by greglandrum — CHANGES_REQUESTED** [review]

File: Docs/Book/Install.md (hunk 1 above)
Comment (UnixJunkie) [reply Docs/Book/Install.md:48]: Will people be able to draw molecules if you remove cairo?
I can add some more text, sure.

Comment (greglandrum) [reply Docs/Book/Install.md:48]: this won't remove cairo. The rdkit package has a cairo dependency, so it installs cairo itself.

**Comment by UnixJunkie** [comment]
superseded by https://github.com/rdkit/rdkit/pull/3593

**Pull request closed without merging**